### DISCUSSION TITLE
Show import/export beyond 24 hours?

### DISCUSSION THREAD

**Discussion by gordonmcdowell** [electricitymaps/electricitymaps-contrib#6152] [Ideas]
It would be EXTREMELY helpful to be able to review import/export as is already shown clearly in last-24-hours perspective, also in the days /months /years perspectives.

(If this sort of thing is best asked on Twitter, or via EM-website-contact-us please let me know.)

**Comment by VIKTORVAV99** [comment]
Hi!

This medium works perfectly (and is preferred over twitter) so no need to worry about that!

This is something we are planning on doing but just haven't been a priority vs other features yet.
Mostly because it involves bigger changes than it may seem from the beginning.

But we are happy to hear your thoughts on the matter (we do use community feedback to set priorities)!
